src/components/shared/PieChartContainer.js

```diff
@@ -132,6 +136,7 @@ class PieChartContainer extends Component {
               fill="#8884d8"
               onMouseEnter={this.onPieEnter}
               {...otherProps}
+              label={'hello'}
```

Comment:
?

Reply:
@akshatnitd @fragm3 Fixed. This was done while trying to debug the label issue. Will try to avoid things .